BEGINNERS.md

```diff
@@ -0,0 +1,291 @@
+# Minimal Snowdrift set up instructions
+
+This is a small tutorial on contributing to Snowdrift. You don't have to
+be tech savvy, but it helps. This is a minimal-ish description of what
+you need to do to get Snowdrift running locally and to start making and
+submitting updates.
+
+Although we recommend deeper study to understand how all of this works,
+you *can* simply follow the steps below, and it will work well enough
+to get started.
+
+Advanced users should adapt any of these instructions as seen fit.
+
+These instructions assume you use Debian testing or some variant of Ubuntu.
+Our general [GUIDE](GUIDE.md) has some notes for other systems.
+
+If you need *any* help at any point, come say "hi" at our freenode.net IRC
+channel [#snowdrift](https://snowdrift.coop/p/snowdrift/w/irc).
+We are always happy to assist and answer *any* questions!
+
+## Installing
+
+Open up a terminal, and run these commands, exactly as you see them. You
+could even copy and paste all of them into your terminal at once, and it
+would work.
+
+    sudo aptitude update
+    sudo aptitude install curl git postgresql postgresql-client
+    sudo ln -s /usr/share/x86_64-linux-gnu/libgmp.so.10 /usr/share/x86_64-linux-gnu/libgmp.so
+    mkdir builds
+    cd builds
+    curl -ssL \
+      https://www.haskell.org/ghc/dist/7.8.4/ghc-7.8.4-x86_64-unknown-linux-deb7.tar.xz |
+      tar xJv
+    cd ghc-7.8.4
+    ./configure && make && sudo make install
+    cd ..
+    curl -ssL \
+      https://www.haskell.org/cabal/release/cabal-install-1.22.2.0/cabal-install-1.22.2.0.tar.gz |
+      tar xzv
+    cd cabal-install-1.22.2.0
+    sudo ./bootstrap.sh
```

Comment:
ah, indeed. We had that in the original GUIDE… @pharpend suggested it would be automatically pulled in or something, but we can add it to the installed stuff earlier